src/api/app/controllers/webui/request_controller.rb

```diff
@@ -110,6 +110,23 @@ def show
     @history = History.find_by_request(@bsreq, {withreviews: 1})
     @actions = @req['actions']
 
+    # retrieve a list of all package maintainers what are assigned to at least one target package
+    @package_maintainers = get_target_package_maintainers(@actions)
+
+    # search for a project, where the user is not a package maintainer but a project maintainer and show
+    # a hint if that package has some package maintainers (issue#1970)
+    @show_project_maintainer_hint = false
+
+    unless @package_maintainers.empty?
```

Comment:
This lines (118-128) now can be:
```ruby
@show_project_maintainer_hint = !@package_maintainers.include?(User.current) && @actions.uniq{|action| action[:tprj]}.any?{|action| User.current.is_maintainer_of_project?(action[:tprj])}
```
I added a call to `uniq` before the `any?` to remove duplicated calls to `User.current.is_maintainer_of_project?` with the same project, so it will be cheaper